www-client/otter-browser/otter_browser-0.9.12.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A lightweight QtWebKit browser"
+DESCRIPTION="Project aiming to recreate the best aspects of the \
+classic Opera (12.x) UI using Qt5."
+HOMEPAGE="http://otter-browser.org/"
```

Comment:
https://otter-browser.org/ is available through https (with Let's Encrypt certificates) so you might wish to change HOMEPAGE to:
```
HOMEPAGE="https://otter-browser.org/"
```